test/helm-test-e2e.sh

```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/env bash
+
+set -o errexit
+set -o nounset
+set -o pipefail
+set -o xtrace
+
+# Install and initialize helm/tiller
+HELM_URL=https://storage.googleapis.com/kubernetes-helm
+HELM_TARBALL=helm-v2.2.3-linux-amd64.tar.gz
```

Comment:
We're currently testing against Helm 2.1.3 in the CI here, we will likely move over to 2.2.3 soon.